api/envoy/api/v2/core/protocol.proto

```diff
@@ -112,8 +112,18 @@ message Http1ProtocolOptions {
   bool enable_trailers = 5;
 }
 
-// [#next-free-field: 13]
+// [#next-free-field: 14]
 message Http2ProtocolOptions {
+  // Defines a parameter to be sent in the SETTINGS frame.
+  // See `RFC7540, sec. 6.5.1 <https://tools.ietf.org/html/rfc7540#section-6.5.1>`_ for details.
+  message SettingsParameter {
+    // The 16 bit parameter identifier.
+    google.protobuf.UInt32Value identifier = 1 [(validate.rules).uint32 = {lte: 65536 gte: 1}];
```

Comment:
Is this intended for setting just the standardized options? Or experimental as well? It is unclear from the comment below. If just for standardized options should we make the `identifier` enum?

Reply:
This is not just for standardized options but any new options (whether RFC bound or experimental).